aten/src/ATen/native/native_functions.yaml

```diff
@@ -2077,6 +2077,14 @@
     Meta: empty_strided_meta
     QuantizedCPU, QuantizedCUDA: empty_strided_unknown_quantized
 
+- func: empty_strided.SymInt(SymInt[] size, SymInt[] stride, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
+  dispatch:
+    CPU: empty_strided_symint_cpu
+    CUDA: empty_strided_symint_cuda
+    MPS: empty_strided_symint_mps
+    Meta: empty_strided_symint_meta
+    QuantizedCPU, QuantizedCUDA: empty_strided_symint_unknown_quantized
```

Comment:
Now that https://github.com/pytorch/pytorch/pull/82189 is in it's not necessary to dupe the dispatch table here. @Krovatkin

Reply:
hmmm why is that? It seems like in master today, we still dupe the dispatch table for `empty.memory_format` and `empty.SymInt`.

Reply:
yeah we haven't fixed it yet